pkgs/development/libraries/vapoursynth/default.nix

```diff
@@ -11,18 +12,18 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "vapoursynth-${version}";
-  version = "R38";
+  version = "R39";
 
   src = fetchFromGitHub {
     owner  = "vapoursynth";
     repo   = "vapoursynth";
     rev    = version;
-    sha256 = "0nabl6949s7awy7rnr4ck52v50xr0hwr280fyzsqixgp8w369jn0";
+    sha256 = "0cw7w8xiwhxhwykydy13m44wm9vn9hrsi30z6017ngga9d84fhqy";
   };
 
   nativeBuildInputs = [ pkgconfig autoreconfHook ];
   buildInputs = [
-    zimg libass tesseract yasm
+    zimg libass tesseract yasm nasm
```

Comment:
Are both yasm and nasm are needed, and are not they used as programs and should be listed in `nativeBuildInputs`?

Reply:
Thanks. No, apparently they switched to nasm and I forgot to remove yasm.